### DISCUSSION TITLE
How to set up the velocities in the SCC MD in DFTB+?

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1466] [Q&A]
Hello everyone,

I am trying to perform a MD simulation for a large system using DFTB+ code. I was preparing the input file

My input file looks like this:

Geometry = VaspFormat {
  <<< "POSCAR"
}

Driver = VelocityVerlet {
  MovedAtoms = 1:-1
  Steps = 20000        # Number of MD steps
  TimeStep [fs] = 1.0       # Time step in femtoseconds
  Thermostat = NoseHoover {
    Temperature [Kelvin] = 300.0  # Temperature in Kelvin
    CouplingStrength [cm^-1] = 3200
  }
  OutputPrefix = "md"
  MDRestartFrequency = 1   # Frequency of saving restart files

  Velocities [Angstrom/fs] = Yes
  WriteXYZ = Yes   # Write atomic positions to XYZ file at each step
}

Hamiltonian = DFTB {
  SCC = Yes
  SCCTolerance = 1e-6
  MaxSCCIterations = 1000
  SlaterKosterFiles = Type2FileNames {
    Prefix = "/glfs/brick01/gv0/girilab/Achyut/Potential/skfiles/matsci-0-3/"
    Separator = "-"
    Suffix = ".skf"
  }
   MaxAngularMomentum = {
    C = "p"
    H = "s"
    B = "p"
    O = "p"
	}
	ForveEvaluation = Dynamics
  Eigensolver = RelativelyRobust {}
  Filling = Fermi {
    Temperature [Kelvin] = 300.0
  }
}

Options {
	RandomSeed = 3871906
	WriteResultsTag = Yes
}

Analysis {
  CalculateForces = Yes
  CalculateStress = Yes
}

ParserOptions {
  ParserVersion = 14
}
Parallel = {
  UseOmpThreads = Yes
}

I want to see how the output file looks like for the MD calculation in DFTB+

Does it give the forces, cell parameters, virials and energy at each MD step?

When I tried to run the above code, it displayed that KpointsAndWeights were missing but in the recipee, I didnt see any K points mentioned in the MD calculation.

Can anyone give feedback on improving this input file for getting the forces, energy, atomic positions and stresses at each MD calculation?

I look forward to listening from any experts in the field.'

Best Regards
Achyut

**Comment by aradi** [comment]
If you have a periodic system, you need to specify K-points.
